### DISCUSSION TITLE
YouTube Playlist

### DISCUSSION THREAD

**Discussion by sagarh1** [sayaka-sh/spmp#388] [Q&A] [answered]
How can I upload my YouTube playlist to the SpMp app...

**Comment by sayaka-sh** [comment]
I'm not sure what you mean. Could you be more specific?

**Reply by sagarh1** [reply]
How to add my YouTube Playlist in your app?

**Reply by sayaka-sh** [reply, marked as answer]
You don't really need to add anything, just sign in and access it via the playlists page in the library

**Comment by sagarh1** [comment]
Songs are not playing...only song loading indicators appear.I reinstalled the application, but it still does not function.

**Reply by sayaka-sh** [reply]
See the pinned issue

**Reply by sagarh1** [reply]
Hii dev,
I tried release.apk and debug.apk, but both apks have the same problem—they take a very long time to load.You will address this problem in the next version as soon as you can.❤️![Screenshot_2025-03-01-00-29-20-728_com.toasterofbread.spmp.debug.jpg](https://github.com/user-attachments/assets/7ab4858b-9c64-4609-a70b-1504f8172d87)
